app/src/main/java/com/projectfkklp/saristorepos/activities/dashboard/DashboardPage.java

```diff
@@ -21,12 +27,19 @@ public class DashboardPage extends AppCompatActivity {
     DashboardSalesLineChart analyticsChart;
     DashboardTopPieChart topSellingChart;
     DashboardTopPieChart topSoldChart;
+    DashboardTopPieChart todaySalesChart;
+
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.dashboard_page);
 
+        // Example values, Today Sales Data
+        int overallSalesTarget = 5000;
+        int actualSales = 3200;
+        int incomeFromYesterday = 10;
+
```

Comment:
- This should be pass to initializer function of DashboardTodaySalesChart for cleaner code
```java
todaySalesChart.setData(todayActualSales, todayTargetSales, yesterdaySales)
```
- Also, please redefine these variables as
-- todayActualSales
-- todayTargetSales
-- yesterdaySales
- from raw variable, we can calculate the:
- - remainingTargetSales - the grey area of pie
- - remainingTargetSalesPercent - it's the 64% to  goal
- - growthRatePercent = (todayActualSales-yesterdaySales)/yesterdaySales